client/dist/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <body style = 'background: #E9D3B3'>
+    </body>
+    <div id = 'app'></div>
+    <script src= 'bundle.js'></script>
+  </head>
+</html>
```

Comment:
No biggie but add a newline at the end of files. I wouldn't go back and change it right now but whenever you remember in the future.